src/lsm/lsm_manager.c

```diff
@@ -458,7 +418,7 @@ __lsm_manager_run_server(WT_SESSION_IMPL *session)
 			    lsm_tree->nchunks > 1) ||
 			    (lsm_tree->queue_ref == 0 &&
 			    lsm_tree->nchunks > 1) ||
-			    (lsm_tree->merge_aggressiveness > 3 &&
+			    (lsm_tree->merge_aggressiveness > 2 &&
```

Comment:
Should this `2` be `WT_LSM_AGGRESSIVE_THRESHOLD`?  Are these numbers related?  If not, maybe another macro value that explains what this number is?